compiler/modules/bir/verify.bal

```diff
@@ -65,11 +65,242 @@ class VerifyContext {
     }
 }
 
+type RegSet boolean[];
+type BlockSet boolean[];
+
+// The set of registers that are initialized via a given edge starting at `origin` in to a block
+type RegFlow record {|
+    Label? origin; // origin == () iff entry
+    RegSet regs;
+|};
+
 public function verifyFunctionCode(Module mod, FunctionDefn defn, FunctionCode code) returns Error? {
-    VerifyContext cx = new(mod, defn);
-    foreach BasicBlock b in code.blocks {
-        check verifyBasicBlock(cx, b);
+    VerifyContext vc = new(mod, defn);
+    Position funcPos = defn.position;
```

Comment:
Why are we passing this around rather than getting it from `vc`?

Reply:
For `verifyGraph` and `verifyRegFlow`, we are not passing around `funcPos`, it gets passed in as the boundary condition, but subsequent recursive calls get a different value.

I can get it via `vc` for `verifyReachable`, but though it's not worth it since I'll have to add a method to `vc`.